capitalize_decorator.rb

```diff
@@ -0,0 +1,8 @@
+require './decorator'
+
+# Class for name captalization
+class CapitalizeDecorator < Decorator
+  def correct_name
+    super.capitalize
+  end
```

Comment:
- [ ] It's expected to Capitalize the first letter of the given name. Let's implement that as done in the code below so the the first letter of the name given is capitalized 👍🏽

```ruby   
   def correct_name
      @nameable.correct_name.capitalize
    end
```